mlflow/keras.py

```diff
@@ -378,7 +378,14 @@ def on_train_end(self, logs=None):
                 sum_list = []
                 self.model.summary(print_fn=sum_list.append)
                 summary = '\n'.join(sum_list)
+            except mlflow.exceptions.MlflowException as e:
```

Comment:
To make this less verbose & more standardized, we could add a helper function that takes a lambda that performs our logging & does the try-catch, e.g.

```
def try_mlflow_log(fn):
  try:
    fn()
  except mlflow.exceptions.MlflowException as e:
    warnings.warn("Logging to Mlflow failed: " + str(e))
```

and then wrap our logging calls in e.g. `try_mlflow_log(fn=lambda: mlflow.log_param("blah", "blah")`

Reply:
Also, maybe we should just broadly catch exceptions here to be safe (i.e. `except Exception:  # pylint: disable=broad-except` instead of `except mlflow.exceptions.MlflowException as e`